Core/Tools/Autorun/Jsupport.cpp

```diff
@@ -20,14 +20,14 @@
 // DBCS Support Codes
 
 #include <windows.h>
-#include "JSUPPORT.H"
+#include "JSUPPORT.h"
 
-// �O�u�֑�����
+// �O�u�֑�����
 // Can't set these characters on top of line
 static BOOL IsDBCSInvalidAtTop(unsigned int c)
 {
-	static BYTE * dtbl = (BYTE *)"�����f�h�񁌁����A�B�X�r�t�v�x�z�l�����������������J�K�T�U�@�B�D�F�H�b�������������E�[�R�S�I���j�C�D�F�G�H�n�p";
-	static BYTE * stbl = (BYTE *)"!%),.:;?]}������";
+	static BYTE * dtbl = (BYTE *)"�����f�h�񁌁����A�B�X�r�t�v�x�z�l�����������������J�K�T�U�@�B�D�F�H�b�������������E�[�R�S�I���j�C�D�F�G�H�n�p";
```

Comment:
Something went wrong here.

Reply:
Yikes - restored and saved again, it's fixed